.github/ISSUE_TEMPLATE/epic-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Epic
+about: This template provides a basic structure for an epic issues.
+title: '[Epic] <title>'
+labels: Epic
+---
+
+### Goal
+
+_Introduction_
+
+### Success Metrics
+
+### Work items
+
+#### Design and specification 📔
+
+- [ ] Design Document
+- [ ] Architectual Decisions Document
+- [ ] Integration Test Complete
+
+#### Engineering ⚙️
+
+- [ ] <User Stories>
+
+#### Release planning
+
+- [ ] Demo the feature internally
+- [ ] GSF updates added to code
```

Comment:
Can you elaborate on this a bit?  What updates do you envision being added before we make a PR to GSF?